src/integrationtest/resources/features/query.feature

```diff
@@ -1,29 +1,20 @@
 @QueryFeature
 Feature: Query test
 
-  @QueryMissedNotificationsScenario
-  Scenario: Query missed notifications
-    Given frontend is up and running
-    When a 'GET' request is prepared for REST API '/failed-notifications'
-    And param key 'subscriptionName' with value 'NonExistingSubscription' is added
-    And request is sent
-    Then response code 200 is received
-    And response body '{"queryResponseEntity":{}}' is received
-
   @QueryByIdScenario
   Scenario: Query aggregated object by ID
     Given frontend is up and running
     And an aggregated object is created
-    When a 'GET' request is prepared for REST API '/queryAggregatedObject'
-    And param key 'ID' with value '6acc3c87-75e0-4b6d-88f5-b1a5d4e62b43' is added
+    When a 'GET' request is prepared for REST API '/aggregated-objects'
```

Comment:
please rename the query.feature file to aggregated-objects

Reply:
Forgot about that. Changed name on the runner as well.